_posts/2021-3-25-Completing-Game-One-From-The-Video.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Completing Game One From The Video
+description: 
+summary: 
+tags: 
+---
+Today, we began work with continuing our progress on Typing Club. After, we sent Kajari the results and she gave us a new task. The task revolves around a youtube video based on coding 7 different games to help us understand more of the variety of games to code. [Here is the video we are currently using for the tasks](https://www.youtube.com/watch?v=lhNdUVh3qCc) The video gives explanations and goes step by step on certain coding elements in order for the specific game to function. The first game we worked on is the Memory Game. I created a new folder titled Memory_Game, and added the specific files to commit the game to work. I than created an Images folder for the game and added the images provided to us by the instructor of the video. I added the html page layout and began copying the instructor's code. The instructor would teach us of what certain elements do, and how the coding functions.
```

Comment:
The instructor would teach us of what certain elements do -- what are these elements? List them here and explain what each does.